LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_LED_PWM.TcPOU

```diff
@@ -0,0 +1,56 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="FB_LED_PWM" Id="{5fae4b24-b1d6-4ac5-a010-9aff4e2893db}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_LED_PWM
+(*
+    Reads an percentage of illumination determined by the user and converts it to a raw value
+    to send to the terminal output via the FB_AnalogOutput.
+    2022-5-03 Maarten Thomas-Bosum
+*)
```

Comment:
I think some of the PR text should make it into the documentation comment here- assumptions made, devices used on, etc.